### ISSUE TITLE
Reason for using large content-weight and style-weight?

### ISSUE THREAD

**Issue by vinsis** [kewellcjj/pytorch_style_transfer#2]
Hi, can you please tell why you are using such high values for `content-weight` and `style-weight` arguments [here](https://github.com/kewellcjj/pytorch-multiple-style-transfer/blob/master/neural_style.py#L185-L188)?

**Comment by kewellcjj** [comment]
@vinsis The weights I used are directly from the cited [repo](https://github.com/pytorch/examples/tree/master/fast_neural_style).  If I understand the loss function correctly, there are no absolute high or low weights, but relative high/low between weights and the learning rate. In other words, one can have even higher weights to get the same results as long as you adjust your learning rate accordingly. Here the learning rate is 0.001.

**Comment by vinsis** [comment]
Got it, thanks.
